src/bin/pgcopydb/copydb.c

```diff
@@ -803,6 +793,12 @@ copydb_init_table_specs(CopyTableDataSpec *tableSpecs,
 		.indexSemaphore = &(specs->indexSemaphore)
 	};
 
+	if (tmpTableSpecs.source_pguri == NULL || tmpTableSpecs.target_pguri == NULL)
+	{
+		log_error(ALLOCATION_FAILED_ERROR);
+		return false;
+	}
+
```

Comment:
Same, the allocation error was so far back that I would prefer to not have this check here.